README.md

```diff
@@ -129,51 +129,89 @@ the `encryption_*` and `ssl_*` options
 directly to the constructor as keyword arguments, even though it is
 required when they are placed in the environment file.
 
-Creating PAM or Native Credentials File (.irodsA)
--------------------------------------------------
+Creating a PAM or Native Authentication File
+--------------------------------------------
 
-Two free functions exist for creating encoded authentication files:
+The following free functions may be used to create the authentication secrets files (called
+`.irodsA` per the convention of iRODS's iCommands):
+   - `irods.client_init.write_native_irodsA_file`
+   - `irods.client_init.write_pam_irodsA_file`
+
+These functions can roughly be described as duplicating the function of `iinit`,
+provided that a valid `irods_environment.json` has already been created.
```

Comment:
Apologies for the pedantry...

`iinit` is also used to "initialize" a client environment from scratch, asking the user a series of questions such as username, hostname/port of the server with which to connect, and zone name, and then generating an `irods_environment.json` file. My understanding is that the client environment already needs to be set up one way or another in order to use this.

So, this is really duplicating the "authentication" functionality of `iinit`. Not sure if that's even worth communicating here, but just wanted to point out that it's not a *complete* replacement for `iinit`, technically.

Reply:
done!